tests/RobotFramework/tests/installation/install_tedge.robot

```diff
@@ -9,7 +9,7 @@ Test Teardown    Get Logs
 *** Test Cases ***
 Install latest via script (from current branch)
     Execute Command    curl -fsSL https://thin-edge.io/install.sh | sh -s
-    Tedge Version Should Match Regex    ^\\d+\\.\\d+\\.\\d+$
+    Tedge Version Should Match Regex    ^\\d+\\.\\d+\\.\\d+(-rc\\.\d+)?$
```

Comment:
I think you're missing an escape char `\\d+` instead of `\d+`

Reply:
Sod it, I'd just corrected that from when I initially missed the `\` entirely!